schema/src/project.yaml

```diff
@@ -70,11 +139,143 @@ slots:
   project_developer:
     range: ProjectRole
     inlined: false
+  id:
+    range: uriorcurie
+    identifier: true
+  image:
+    slot_uri: schema:image
+    description: an image.
+  address:
+    slot_uri: schema:address
+    description: an address.
+  showOnProjectPage:
+    range: boolean
+    description: Whether to show this organization on the project page.
+  marketType:
+    slot_uri: rfs:marketType
+    range: MarketTypeTypes
+    description: The type of market for the associated credits.
+  bioregion:
+    range: string
+    slot_uri: rfs:bioregion
+    description: The bioregion associated with the project.
+  biomeType:
+    range: string
+    slot_uri: rfs:biomeType
+    description: The type of biome associated with the project.
+  watershed:
+    range: string
+    slot_uri: rfs:watershed
+    description: The watershed associated with the project.
+  subWatershed:
+    range: string
+    slot_uri: rfs:subWatershed
+    description: The sub-watershed associated with the project.
+  ecosystemType:
+    slot_uri: rfs:ecosystemType
+    range: EnvironmentTypeTypes
+    description: The type of ecosystem associated with the project.
+    multivalued: true
```

Comment:
Are we intentionally using "ecosystem" and "environment type" terms in the same context here?

The current description for the Environment Type taxonomy says it should not be used to describe ecosystems: https://framework.regen.network/taxonomy/environment-type

> The classification is designed to provide a mental image of the landscape where project activities occur. These should not be rigorously defined technical definitions that would be used to describe ecosystems or land cover/land use mapping classes.

Reply:
Ah, I thought that we were already using `ecosystemType` in metadata elsewhere (project? credit class? having trouble finding it now) and that Ned has advised us to change our language from `ecosystem` to `environment`. So my intention here was to stay consistent with existing data, while referencing the data standards work. Do you have perspective or is there someone else we need to rope in?

Reply:
@S4mmyb do you have thoughts here?

Summary is that in the existing old project metadata we used to use `ecosystemType`, but with Ned's work I think there was a desire to switch to `environmentType`, is that correct? Should we be changing all metadata for new projects to use `environmentType` instead of `ecosystemType` now?

Reply:
Yeah it should be `environmentType` as opposed to `ecosystemType`. In general, whatever Ned had in the PR he made we should adopt and overwrite. At some point, we'll have to go back and fix all of the other metadata sets, but we can do that in the future I think. Unless it breaks the way the front end consumes this kind of data. @blushi if we used `environmentType` instead, would that affect our ability to render tags on the pages? If so we probably want to push this change to a future PR where we change everything across the board...

Reply:
We can also have a front end PR that looks for `environmentType` and if not present, falls back to `ecosystemType`, this way we can take our time migrating legacy projects and that won't block us getting these new projects migrated to the new vocabulary.

Reply:
> We can also have a front end PR that looks for `environmentType` and if not present, falls back to `ecosystemType`, this way we can take our time migrating legacy projects and that won't block us getting these new projects migrated to the new vocabulary.

yes we can do what @clevinson proposes